frontend/public/components/monitoring/_monitoring.scss

```
@@ -437,6 +437,10 @@ $monitoring-line-height: 18px;
   }
 }
 
+.query-browser__series-select-all-btn {
+  padding-top: 15px !important;
```

Maybe a standard [Patternfly spacing](https://www.patternfly.org/v4/utilities/spacing#size-values) margin value will suit better this purpose
```suggestion
  margin-top: var(--pf-global--spacer--md);
```

---

👍 Updated in the latest commit : https://github.com/openshift/console/blob/c5e2e41052f4a74edbd4841b5369e458843965d1/frontend/public/components/monitoring/_monitoring.scss#L499

---

"I noticed the UI jumps when deselecting all the series, maybe can be easily fixed matching the height of the "no datapoints found" box with the chart height" 
^^ 👍 Updated in the latest commit : 
https://github.com/openshift/console/blob/c5e2e41052f4a74edbd4841b5369e458843965d1/frontend/public/components/monitoring/_monitoring.scss#L597